code/game/machinery/doors/door.dm

```diff
@@ -253,7 +253,13 @@
 	return src.attack_hand(user)
 
 /obj/machinery/door/attack_hand(mob/user as mob)
-	return src.attackby(user, user)

```

Comment:
This seems to be cutting out some of the functionality present in attack_hand, namely lines 342, and 353

Reply:
Should be fixed now.